### PR TITLE
Docker fix

### PR DESCRIPTION
**Pull request by bob0005**
DEV Mode ok, PROD Mode not migrating DB correctly.

### CODE REVIEW

**Comment by bob0005** [comment]
Test comment

**Comment by bob0005** [comment]
Ready for review

File: transcendence.sh
Comment (ElMariuso) [inline transcendence.sh:1]: I think a command in the shell script that does both cleans and removes would be a big plus if you want to remove all.

File: docs/Docker/Dockerfile-backend.txt [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+				This step defines a non-root user (node) to run the application for enhanced security.
+
+			.Launching in Development Mode: "ENV NODE_ENV development
+											 EXPOSE 3000
+											 CMD ["dumb-init", "npm", "run", "start:dev"]"
+				Finally, we set the NODE_ENV to development, 
+				expose port 3000 for the application, 
+				and define the command to start the application in development mode using npm run start:dev
+
+	Development Environment (Dockerfile.prod):
```
Comment (ElMariuso) [inline docs/Docker/Dockerfile-backend.txt]: It is also said here that it is a development environment yet here it is the production environment.

File: README.md
Comment (ElMariuso) [inline README.md:1]: A prerequisite should be added I think. Like you need a bash environment to run the launch script.

**Review by ElMariuso — CHANGES_REQUESTED** [review]
- My backend container is not launching and restart ad vitam aeternam

> $ docker ps
CONTAINER ID   IMAGE                           COMMAND                  CREATED              STATUS                          PORTS                              NAMES
8b1ff528da83   ft_transcendence-frontend-dev   "docker-entrypoint.s…"   About a minute ago   Up About a minute               3000/tcp, 0.0.0.0:8080->8080/tcp   frontend-dev
8070fef50ebf   ft_transcendence-backend-dev    "/usr/src/app/entryp…"   About a minute ago   Restarting (1) 24 seconds ago                                      backend-dev
2860d721e733   postgres:13                     "docker-entrypoint.s…"   About a minute ago   Up About a minute               0.0.0.0:5432->5432/tcp             postgres

> $ docker logs backend-dev
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory
exec /usr/src/app/entrypoint.sh: no such file or directory

- The frontend launches but leads to an error page.

image

**Pull request merged**